content/events/bmt-2024.md

```diff
@@ -0,0 +1,333 @@
+| 10/15 – 10/26 | Adjustment          | $25               | Shirts are not guaranteed, but may be purchased day-of for $10.                        |
+
+\*All dates are in US Pacific Time. That is, Phase 2 registration ends at 11:59
+PM PT on 10/14/2024.
+
+\*\*For students facing financial hardship, please contact <team@berkeley.mt>
+for a fee waiver.
+
+## How to Register
+
+Most pre- and post-contest operations, including registration, team formation,
+waivers, and score reports, should be performed on
+[ContestDojo](https://contestdojo.com/), an online math competition platform.
+For people unfamiliar with ContestDojo, we have a released a guide, which can be
+found here: [ContestDojo Guide].
+
+**All contestants are required to have their parent/guardian sign UC Berkeley’s
+consent waivers through this portal. Contestants will NOT be allowed to take the
+contest without having a signed liability and consent to treatment waiver.**
+
+Any questions about the ContestDojo portal should be directed to us at
+<team@berkeley.mt>.
+
+## Student Eligibility
+
+- Students must be in **grade 12 or below** to be eligible for BMT.
+- Students do not have to be local to the California Bay Area or the U.S. to
+  participate in BMT. However, we will offer BMT Online in either December or
+  January (TBA), which may be a better fit for some students.
+- Coaches may register multiple teams. We ask that **for every 2 teams, there is
+  at least 1 parent chaperone**.
+
+## Team Finder
+
+Students may register as a team of 1, however, it is beneficial to compete on a
+team of at least 4 due to our scoring guidelines. For students who do not have a
+full team, we organized a [Team-Finder] document. Follow all outlined steps in
+the document.
+
+## Event Schedule
+
+Our event schedule is yet to be finalized, but we expect the tournament to start
+at 7:30 AM and end before 7:30 PM, with a lunch break in the middle.
+
+## Contest Format
+
+### Power Round (90 mins)
+
+The Power Round is a **proof-based team round.** Contestants will answer a
+series of questions centered around a single topic not typically seen in
+competitive mathematics, culminating in incredible results in the field of that
+chosen topic. For all questions (unless stated otherwise), teams will need to
+provide a rigorous mathematical justification for their assertions in full
+sentences.
+
+The topic for the Power Round will likely be at an undergraduate level, but no
+substantial previous mathematical background is expected.
+
+### Individual Round
+
+For the Individual Round, students may choose to take either the General Test
+or two Focus Tests.
+
+#### General Test (90 mins)
+
+The General Test is an individual round consisting of questions across the
+various subjects that comprise the focus tests. This round is highly recommended
+for students with less prior contest experience.
+
+#### Focus Tests (60 mins)
+
+There are four focus tests of 10 questions across four subjects --- Algebra,
```

Comment:
This is not how to make an emdash in markdown — you want to put the actual character there